tests/sles4sap/hana_cluster.pm

```diff
@@ -90,15 +117,15 @@ sub run {
             # sr_state is not online after 90 seconds. Start system again and retry
             assert_script_run $start_cmd;
             sleep bmwqemu::scale_timeout(10);
-            assert_script_run "until su - $sapadm -c 'hdbnsutil -sr_state' | grep -q 'online: true' ; do sleep 1 ; done";
+            assert_script_run "until su - ${sapadm} -c 'hdbnsutil -sr_state' | grep -q 'online: true' ; do sleep 1 ; done";
         }
 
         # Synchronize the nodes
-        barrier_wait "HANA_CREATED_CONF_$cluster_name";
+        barrier_wait "HANA_CREATED_CONF_${cluster_name}";
```

Comment:
Added `{}` again.

Reply:
Curly braced variable is safely enclosed variable.